### ISSUE TITLE
setApprovalForAll via OpenSea website will trigger Metamask error 

### ISSUE THREAD

**Issue by zhangzhongnan928** [SmartTokenLabs/ST404#4]
1. list a STTT on OpenSea
2. Opensea will ask you to setApprovalForAll
3. Metamask shows error 

Tried JoyID, works fine.

**Comment by oleggrib** [comment]
Hi, @zhangzhongnan928  , its known Metamask bug, which will be fixed in the next Metamask release
https://github.com/MetaMask/metamask-extension/issues/18507
Same issue exist for defeult ERC404v2

**Comment by SmartLayer** [comment]
> Hi, @zhangzhongnan928 , its known Metamask bug, which will be fixed in the next Metamask release [MetaMask/metamask-extension#18507](https://github.com/MetaMask/metamask-extension/issues/18507) Same issue exist for defeult ERC404v2


The issue reported on GitHub regarding MetaMask's setApprovalForAll function primarily affects MetaMask on Chrome. Users did not encounter this error on mobile MetaMask, Firefox, and Brave browsers, so we ought to try that. Other than that, @zhangzhongnan928 can try doing that on WriteContract to bypass this and further the test.

**Comment by Fast288** [comment]
hi, i still have this issue with forge token. ca: 0x73576a927cd93a578a9dfd61c75671d97c779da7
Can't list it on opensea.
how can i fix it?

**Comment by oleggrib** [comment]
@Fast288 , are you sure it's correct repo for 0x73576a927cd93a578a9dfd61c75671d97c779da7 ?

**Comment by Fast288** [comment]
yes, its the token contract adress: 0x73576A927Cd93a578a9dFD61c75671D97c779da7
FORGE token

its ERC404.

I cant list it to on opensea, my metamask gives me the error.

**Comment by oleggrib** [comment]
@Fast288 , we didn't deploy this contract to mainnet, so you are wrong. Contract name is ST404, not ERC404

**Comment by Fast288** [comment]
ok, any idea how i can fix the issue?
friends of me don't have this issue and they're also using Metamask

**Comment by SmartLayer** [comment]
I am trying to redo the process. note that on the test net, the lastest deployment is https://testnets.opensea.io/collection/st404-testing-token-v3-4/

**Comment by SmartLayer** [comment]
> Hi, any update on this? how can i get around this problem?

You are using $FORGE token, which is not this project:

> 0x73576A927Cd93a578a9dFD61c75671D97c779da7

I followed up your post when you posted to https://github.com/MetaMask/metamask-extension/issues/18507
